database/tables/group_user_roles.pg

```diff
@@ -0,0 +1,17 @@
+columns
+    group_id: bigint not null
+    group_role_id: bigint not null
+    id: bigint not null default nextval('group_user_roles_id_seq'::regclass)
+    user_id: bigint not null
+
+indexes
+    group_user_roles_pkey: PRIMARY KEY (id) USING btree (id)
+    group_user_roles_group_id_user_id_group_role_id_key: UNIQUE USING btree (group_id, user_id, group_role_id)
+    group_user_roles_group_id_key: USING btree (group_id)
+    group_user_roles_group_role_id_key: USING btree (group_role_id)
+    group_user_roles_user_id_key: USING btree (user_id)
```

Comment:
Are all these indexes actually necessary? AFAICT, every query of this table always specifies `group_id`, so I suspect that the `UNIQUE` index alone will get us the performance we want.